zboss/README.rst

```diff
@@ -12,7 +12,6 @@ The nrfxlib repository contains the following versions of the ZBOSS libraries:
   This version is enabled by default.
 
   The production libraries fully conform to the certification, but they are not necessarily certified.
-  See the compatibility matrices in the `Infocenter`_ for the certification status of the production library that is present in a given |NCS| release.
```

Comment:
This line was introduced in a PR opened and merged when I was on vacation: https://github.com/nrfconnect/sdk-nrfxlib/pull/409

@tomchy , @wbober , any clues why this was added in this form? Infocenter does not contain any information about the certification status of NCS libraries for Zigbee...